src/app/views/sap/create_mission.php

```diff
@@ -15,10 +15,10 @@
                 <?php
                 if (isset($errors) && count($errors) !== 0) {
                     foreach ($errors as $error) {
-                        echo "<p style=\"color:red\">$error</p>";
+                        echo "<p class=\"text-error\">$error</p>";
```

Comment:
I would personally prefer a string concatenation over escaping quotes! But, okay. No problem with this.